codex/blockexchange/network/network.nim

```diff
@@ -290,6 +290,7 @@ proc dialPeer*(b: BlockExcNetwork, peer: PeerRecord) {.async.} =
     trace "Skipping dialing self", peer = peer.peerId
     return
 
+  trace "Dialing peer...", peer = peer.peerId
```

Comment:
libp2p already has the same exact log, but no harm I guess :)

Reply:
Yup found it after you mentioned it in the meeting. I'd rather use the libp2p one.